src/shorts.js

```diff
@@ -0,0 +1,35 @@
+/* eslint no-redeclare: 0 */
+/* global fetch:writable */
+import { configRead } from './config';
+
+const origParse = JSON.parse;
+JSON.parse = function () {
+  const r = origParse.apply(this, arguments);
+  if (!configRead('removeShorts')) {
+    return r;
+  }
+
+  // First page of subscriptions tab
+  const gridRenderer =
+    r?.contents?.tvBrowseRenderer?.content?.tvSecondaryNavRenderer?.sections[0]
+      ?.tvSecondaryNavSectionRenderer?.tabs[0]?.tabRenderer?.content
+      ?.tvSurfaceContentRenderer?.content?.gridRenderer;
```

Comment:
This is a pretty long/specific path, and I don't know how stable it will be. Is there another way to find the target element?

Reply:
I thought about it - doing something like a depth search on the `r` object to find `gridRenderer` would get rid of th hardcoded path and should be better in terms of keeping it up to date, but it'll run on all JSONs that are being parsed possibly leading to longer loading times/stutter.

Reply:
Would it be possible to string search the unparsed JSON for `gridRenderer` and replace it with a misspelling? Would YouTube throw when it encounters unexpected keys?

Reply:
`gridRenderer` contains all cells with both shorts and standard videos, so this approach won't work. You have to look inside each object and remove it from the list.